.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest]
+
+    steps:
+      - name: XCode version
+        run: xcode-select -p
+
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
You don't need 0, something like 100 (and tags if necessary) will likely do, probably better 200 to play it safe. Our total number of commits is close to 10000, but by limiting it to what's likely to happen within a couple of months you still get most of the speed advantages of a shallow clone. (I.e., it'll take ~1 second as opposed to ~4. Maybe not worth the optimization when the build itself takes over 10 minutes, but just fyi anyway. :-P)

Reply:
Happy to change it :+1: